### PR TITLE
DLS-10598 Contact Details Error

### PR DESCRIPTION
**Pull request by LAJW**

### CODE REVIEW

File: app/uk/gov/hmrc/cbcrfrontend/controllers/SharedController.scala [hunk 1]
```diff
@@ -107,10 +107,14 @@ class SharedController @Inject() (
     single("cbcId" -> of[CBCId])
   )
 
-  val technicalDifficulties: Action[AnyContent] = Action { implicit request =>
+  def technicalDifficulties: Action[AnyContent] = Action { implicit request =>
     InternalServerError(views.errorTemplate("Internal Server Error", "Internal Server Error", "Something went wrong"))
   }
 
+  def contactDetailsError: Action[AnyContent] = Action { implicit request =>
```
Comment (BetaDraconis) [inline app/uk/gov/hmrc/cbcrfrontend/controllers/SharedController.scala:114]: See above

File: app/uk/gov/hmrc/cbcrfrontend/controllers/SharedController.scala [hunk 2]
```diff
@@ -126,7 +130,7 @@ class SharedController @Inject() (
   private def cacheSubscriptionDetails(s: SubscriptionDetails, id: CBCId)(implicit hc: HeaderCarrier): Future[Unit] =
     (cache.save(TIN(s.utr.value, "")) *> cache.save(s.businessPartnerRecord) *> cache.save(id)).map(_ => ())
 
-  val submitCBCId: Action[AnyContent] = Action.async { implicit request =>
+  def submitCBCId: Action[AnyContent] = Action.async { implicit request =>
```
Comment (BetaDraconis) [inline app/uk/gov/hmrc/cbcrfrontend/controllers/SharedController.scala:133]: See above

File: app/uk/gov/hmrc/cbcrfrontend/controllers/SubscriptionController.scala [hunk 3]
```diff
@@ -135,26 +135,23 @@ class SubscriptionController @Inject() (
     }
   }
 
-  val updateInfoSubscriber: Action[AnyContent] = Action.async { implicit request =>
+  def updateInfoSubscriber: Action[AnyContent] = Action.async { implicit request =>
```
Comment (BetaDraconis) [inline app/uk/gov/hmrc/cbcrfrontend/controllers/SubscriptionController.scala:138]: if this is a method this should probably have empty parentheses

File: app/uk/gov/hmrc/cbcrfrontend/controllers/SharedController.scala [hunk 4]
```diff
@@ -107,10 +107,14 @@ class SharedController @Inject() (
     single("cbcId" -> of[CBCId])
   )
 
-  val technicalDifficulties: Action[AnyContent] = Action { implicit request =>
+  def technicalDifficulties: Action[AnyContent] = Action { implicit request =>
```
[the remark this answers is not in the document]
Comment (LAJW) [reply app/uk/gov/hmrc/cbcrfrontend/controllers/SharedController.scala:110]: Not necessarily. Play docs recommend leaving parentheses out: 
https://www.playframework.com/modules/scala-0.9/controllers

**Review by BetaDraconis — APPROVED** [review]

**Pull request merged**